docs/dev-tools/zombienet/testing.md

```diff
@@ -3,135 +3,213 @@ title: Testing DSL
+        ```
+
+??? function "`Tracing` - match an array of span names from the supplied traceID"
+
+    === "Syntax"
+
+        - `node-name`: `trace with traceID contains ["name", "name2",...]`
+
+    === "Example"
+
+        ```bash
+
+        alice: trace with traceID 94c1501a78a0d83c498cc92deec264d9 contains ["answer-chunk-request", "answer-chunk-request"]
+        
+        ```
+
+??? function "`Custom JS scripts` - run a custom JS script and assert on the return value"
+
+    === "Syntax"
+
+        - `node-name`: `js-script script_relative_path [return is comparator target_value] [within x seconds]`
+
+    === "Example"
+
+        ```bash
+
+        alice: js-script ./0008-custom.js return is greater than 1 within 200 seconds
+        
+        ```
+
+??? function "`Custom TS scripts` - run a custom TS script and assert on the return value"
+
+    === "Syntax"
+
+        - `node-name`: `ts-script script_relative_path [return is comparator target_value] [within x seconds]`
+
+    === "Example"
+
+        ```bash
+
+        alice: ts-script ./0008-custom-ts.ts return is greater than 1 within 200 seconds
+        
+        ```
+
+??? function "`Backchannel` - wait for a value and register to use"
+
+    === "Syntax"
+
+        - `node-name`: `wait for var name and use as X [within x seconds]`
+
+    === "Example"
+
+        ```bash
+
+        alice: wait for name and use as X within 30 seconds
+        
+        ```
 
 ### Commands
 
 Commands allow interaction with the nodes and can run pre-defined commands or an arbitrary command in the node.
 
-| Syntax                                             | Description                                                                                                                                          |
-| -------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- |
-| node-name `restart [after x seconds]`              | Stop the process and start again after the x amount of seconds or innmediatly                                                                      |
-| node-name `pause`                                  | Pause (SIGSTOP) the process                                                                                                                          |
-| node-name `resume`                                 | Pause (SIGCONT) the process                                                                                                                          |
-| `sleep x`                                          | Sleep the test-runner for x amount of seconds                                                                                                       |
+??? function "`restart` - stop the process and start again after the `X` amount of seconds or immediately"
+
+    === "Syntax"
+
+        - `node-name` `restart [after x seconds]`
```

Comment:
can you make all these one string please? so like `node name restart [after x seconds]`